### ISSUE TITLE
Copyright status for Reference Library Publications

### ISSUE THREAD

**Issue by mxgold** [ycba-cia/blacklight-collections2#225]
For all Ref Lib records, copyright status displayed is currently  the equivalent of unknown / requires further research.  (I assume this is bc Blacklight is looking to Orbis 542 field for this data, since this is how we now handle for Rare Books and Manuscripts Dept material, and if this field is null the default is unknown.)

All Ref Lib publications are cataloged with date and place of publication, so copyright status can be clearly determined and stated.  The "Copyright Status" info currently displayed is therefore incorrect and potentially confusing to patrons.

Options:
*suppress the display of the "Copyright Status" field for Ref Lib publications?
*explore populating the 542 field in bulk?  Unlike with artwork or mss where there can be several unknowns and judgment calls may have to be made, we have clearly stated places and dates of publication - could logic be used to script/populate?
*other?

@KraigBinkowski

**Comment by KraigBinkowski** [comment]
I don't think this field is important enough for ref lib records to warrant populating 542. In this instance I would recommend suppressing the Copyright status for Ref materials

**Comment by yulgit1** [comment]
Per missing 542 subfield l, facets have display of "unknown",  and item page have the "Copyright Status" display of "Copyright Information" with a link to http://hdl.handle.net/10079/c59zwbm.

@mxgold @KraigBinkowski are you suggesting the suppress the facet only or both the facet and the item page?

**Comment by mxgold** [comment]
Although my initial suggestion was suppress from display on the item page, pending @KraigBinkowski confirmation,  I think it makes sense to suppress from the facet results too, since it's the status is incorrect.

**Comment by KraigBinkowski** [comment]
Yes, I would support the suppression all instances of copyright notification for ref materials - display on the item page as well as within the facet results.

**Comment by yulgit1** [comment]
@mxgold @KraigBinkowski - I'm going to backtrack on this, maybe the facet should remain as is. I feel like suppressing the Ref Collection from this facet is a loss of information, and more prone to confusion, than just leaving the refs as unknown but at least counted.

So just suppress "copyright status:unknown" on the item page, but leave facet as is.

**Comment by yulgit1** [comment]
suppressed from item page:
https://github.com/ycba-cia/blacklight-collections2/commit/97818467f73d91c3d66c6f48c9439e40afac9f45

**Comment by KraigBinkowski** [comment]
I don't see the advantage of leaving ref items in the facet counts but am willing to discuss. I might argue that the total facet numbers are misleadingly high in that they include those 37,532 ref books that we designate as "unknown"
